packages/docusaurus-theme-common/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "incremental": true,
     "tsBuildInfoFile": "./lib/.tsbuildinfo",
+    "module": "esnext",
```

Comment:
make sense, does it also solve a bug related to ESBuild or can we add this in a separate PR?

Reply:
Don't remember the context anymore. I remember it's initially added to fix some issue, but maybe the issue is no longer here. However, I think this change is good anyways since the theme-common package is designed for browser to consume.